content/news/2023_10_25_GRASS_GIS_8_3_1_released.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "GRASS GIS 8.3.1 released"
+date: 2023-10-25T12:42:00-03:00
+layout: "news"
+author: GRASS Development Team
+---
+
+### What's new in a nutshell
+
+The GRASS GIS 8.3.1 maintenance release provides more than **60 changes** 
+compared to 8.3.0. This new patch release brings in
+**important fixes and improvements** in GRASS GIS modules and the
+graphical user interface (GUI) which stabilizes the new
+**single window layout** active by default.
+
+Some of the most relevant changes include: fixes for `r.watershed` which
+got partially broken in the 8.3.0 release; and a fix for installing addons
+on MS Windows with `g.extension`.
+
+Translations have been continued in
+[Weblate](https://weblate.osgeo.org/projects/grass-gis/), which
+automatically creates pull requests with the translated chunks.
+We'd like to **thank the translators of all languages** for their
+ongoing support!
+
+<a href="/images/news/grass83_news_screenshot.png">
+  <img src="/images/news/grass83_news_screenshot.png"
```

Comment:
What if we use the one that Michael Barton created for #3129: https://www.dropbox.com/scl/fi/v05vth4lutpuaex5sw2uu/GRASS8.3_thematic.png?rlkey=pnq2wus10336hfdn5fftggwww&dl=0

Reply:
Good idea. Does @cmbarton agree?

Reply:
I am of course happy to have my image used like this. Glad you all like it.